dtls.c

```diff
@@ -4395,8 +4407,8 @@ dtls_handle_message(dtls_context_t *ctx,
       } else {
         dtls_stop_retransmission(ctx, peer);
       }
-      break;
-
+      /* no more valid records after alerts */
+      return 0;
```

Comment:
There is [one case](https://github.com/eclipse/tinydtls/pull/190/commits/62c8c08850df36da7201871eb1e10b2d7c58268b#diff-947f252e5a6242133f45e2963433822253886eb9a6bf4c12c7163ac8e90fe017L4396) where `peer` is currently not set to `NULL` (which is correct, I think). Changing `break` to `return 0` here should not break things, but I wanted to mention this case in order to do a double-check.

Reply:
We can keep the old function for DTLS_ALERT_LEVEL_WARNING. But that would only make a difference, if such a Alert/Warning and additional records are sent within the same UDP message.
I will change it.

Reply:
Correct, and I think that a message that contains additional records after the alert can be treated as broken—why would an application do that?